app-admin/genromfs/genromfs-0.5.2.ebuild

```diff
@@ -7,7 +7,8 @@ inherit toolchain-funcs
 
 DESCRIPTION="Create space-efficient, small, read-only romfs filesystems"
 HOMEPAGE="https://romfs.sourceforge.net/"
-SRC_URI="https://downloads.sourceforge.net/${PN}/${P}.tar.gz"
+SRC_URI="https://github.com/chexum/${PN}/archive/refs/tags/${PV}.tar.gz
+	-> ${P}.gh.tar.gz"
```

Comment:
If you're changing SRC_URI such that the hash/manifest changes, please include a summary of the diff of the tarballs in the commit message, per https://devmanual.gentoo.org/general-concepts/manifest/index.html#updating-manifest-files.

Reply:
Please resubmit. I've reverted it in d5b13a5eb13eb91f544fe00417fa5d1d32673d95.